tiledb/cloud/compute/delayed.py

```diff
@@ -170,6 +173,39 @@ def __call__(self, *args, **kwargs):
         return self
 
 
+class DelayedMultiArrayUDF(DelayedBase):
+    def __init__(
+        self,
+        func: Union[str, Callable, None],
```

Comment:
No need for None in this Union anymore. (Sorry for missing that earlier.)

Reply:
and fyi if you push a change for this I'll merge it immediately assuming at least a couple of the tests finish cleanly because there's no real need to wait for all of them.